Unix/base/batch.c

```diff
@@ -15,7 +15,12 @@
 #define BATCH_PAGE_SIZE ((size_t)(BLOCK_SIZE * 2))
 #define BLOCK_SIZE ((size_t)512)
 
-void* Batch_Get(
+void* __Batch_Get(
+#if defined(CONFIG_ENABLE_DEBUG) && defined(CONFIG_OS_LINUX)
```

Comment:
Do you want to restrict this to LINUX? Does the memory dump code work on other platforms? If so, wouldn't we want this layered on top of that?

Reply:
If you look in pal/palcommon.h, you see that PAL_DumpAllocList() is defined only if USE_ALLOCATOR is defined, and USE_ALLOCATOR is defined if CONFIG_ENABLE_DEBUG and CONFIG_OS_LINUX are both defined.